charts/testkube-api/templates/deployment.yaml

```diff
@@ -48,6 +48,10 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           env:
+            {{- if .Values.global.features.logsV2 }}
```

Comment:
does it need to be in global? looks like it can go under `testkube-api` section of helm charts.

Reply:
yes - it'll be also used to control if logs service will be created later

Reply:
Afaiu logs service will be a separate chart. Why not add a `logs-service` block in `values.yaml` file?
Global variables are applied to all charts, for example, `images` variables - if globals are set, images in all charts for all services will be replaced.

But this sounds like smth that will be related to logs service chart only. WDYT?

Reply:
This feature will be used in 
* API service (two places api + scheduler) - when we split scheduler to seprarate service so in 2
* And in logs service. 
* Also we need to configure based on this variable global testkube chart to enable or disable logs service at all.

I can copy-paste between all these places but not sure if it not introduce issues with syncing in future.